### PR TITLE
Rus translation browse

### PR DESCRIPTION
**Pull request by mamedzuka**
## References
_Add references/links to any related issues or PRs. These may include:_
* Fixes #[issue-number]
* Requires DSpace/DSpace#[pr-number] (if a REST API PR is required to test this)

## Description
Short summary of changes (1-2 sentences).

## Instructions for Reviewers
Please add a more detailed description of the changes made by your PR. At a minimum, providing a bulleted list of changes in your PR is helpful to reviewers.

List of changes in this PR:
* First, ...
* Second, ...

**Include guidance for how to test or review your PR.** This may include: steps to reproduce a bug, screenshots or description of a new feature, or reasons behind specific changes. 

## Checklist
_This checklist provides a reminder of what we are going to look for when reviewing your PR. You need not complete this checklist prior to creating your PR (draft PRs are always welcome). If you are unsure about an item in the checklist, don't hesitate to ask. We're here to help!_

- [ ] My PR is small in size (e.g. less than 1,000 lines of code, not including comments & specs/tests), or I have provided reasons as to why that's not possible.
- [ ] My PR passes [TSLint](https://palantir.github.io/tslint/) validation using `yarn run lint`
- [ ] My PR doesn't introduce circular dependencies
- [ ] My PR includes [TypeDoc](https://typedoc.org/) comments for _all new (or modified) public methods and classes_. It also includes TypeDoc for large or complex private methods.
- [ ] My PR passes all specs/tests and includes new/updated specs or tests based on the [Code Testing Guide](https://wiki.lyrasis.org/display/DSPACE/Code+Testing+Guide).
- [ ] If my PR includes new, third-party dependencies (in `package.json`), I've made sure their licenses align with the [DSpace BSD License](https://github.com/DSpace/DSpace/blob/main/LICENSE) based on the [Licensing of Contributions](https://wiki.lyrasis.org/display/DSPACE/Code+Contribution+Guidelines#CodeContributionGuidelines-LicensingofContributions) documentation.

### CODE REVIEW

File: src/assets/i18n/ru.json5 [hunk 1]
```diff
@@ -1549,257 +1549,208 @@
...
   
   // "browse.comcol.by.author": "By Author",
-  // TODO New key - Add a translation
-  "browse.comcol.by.author": "By Author",
+  "browse.comcol.by.author": "По Автору",
   
   // "browse.comcol.by.dateissued": "By Issue Date",
-  // TODO New key - Add a translation
-  "browse.comcol.by.dateissued": "By Issue Date",
+  "browse.comcol.by.dateissued": "По Дате Выпуска",
```
Comment (apalkoff) [inline src/assets/i18n/ru.json5]: Эээ..в русском языке в заглавиях только первая буква заглавная. Остальные строчные.
```suggestion
  "browse.comcol.by.dateissued": "По дате выпуска",
```

File: src/assets/i18n/ru.json5 [hunk 2]
```diff
@@ -1549,257 +1549,208 @@
...
   
   // "browse.comcol.by.pjtitle": "By Title",
-  // TODO New key - Add a translation
-  "browse.comcol.by.pjtitle": "By Title",
+  "browse.comcol.by.pjtitle": "По Заглавию",
   
   // "browse.comcol.by.rodept": "By Department",
-  // TODO New key - Add a translation
-  "browse.comcol.by.rodept": "By Department",
+  "browse.comcol.by.rodept": "По Организации",
```
Comment (apalkoff) [inline src/assets/i18n/ru.json5]:
```suggestion
  "browse.comcol.by.rodept": "По подразделению",
```

File: src/assets/i18n/ru.json5 [hunk 3]
```diff
@@ -1549,257 +1549,208 @@
...
   
   // "browse.comcol.by.rpdept": "By Department",
-  // TODO New key - Add a translation
-  "browse.comcol.by.rpdept": "By Department",
+  "browse.comcol.by.rpdept": "По Организации",
```
Comment (apalkoff) [inline src/assets/i18n/ru.json5]:
```suggestion
  "browse.comcol.by.rpdept": "По подразделению",
```

File: src/assets/i18n/ru.json5 [hunk 4]
```diff
@@ -1549,257 +1549,208 @@
...
   // "browse.comcol.by.rpname": "By Name",
-  // TODO New key - Add a translation
-  "browse.comcol.by.rpname": "By Name",
+  "browse.comcol.by.rpname": "По Имени",
   
   // "browse.comcol.by.subject": "By Subject",
-  // TODO New key - Add a translation
-  "browse.comcol.by.subject": "By Subject",
+  // по Дисциплине или по Теме исследования?
+  "browse.comcol.by.subject": "По Дисциплине",
```
Comment (apalkoff) [inline src/assets/i18n/ru.json5]: думаю тут имеется в виду «по теме» или «по тематике»
```suggestion
  "browse.comcol.by.subject": "По Дисциплине",
```

File: src/assets/i18n/ru.json5 [hunk 5]
```diff
@@ -1549,257 +1549,208 @@
...
-  // TODO New key - Add a translation
-  "browse.comcol.by.type": "By Type",
+  // по Типу или по Формату?
+  "browse.comcol.by.type": "По Типу",
   
   
   // "browse.comcol.head": "Browse",
-  // TODO New key - Add a translation
-  "browse.comcol.head": "Browse",
+  "browse.comcol.head": "Поиск",
```
Comment (apalkoff) [inline src/assets/i18n/ru.json5:1586]: Мне не нравится, может «Посмотреть»

```suggestion
  "browse.comcol.head": "Посмотреть",
```

File: src/assets/i18n/ru.json5 [hunk 6]
```diff
@@ -1549,257 +1549,208 @@
...
   
   // "browse.empty": "No items to show.",
-  // TODO New key - Add a translation
-  "browse.empty": "No items to show.",
+  "browse.empty": "Данных по запросу не найдено.",
```
Comment (apalkoff) [inline src/assets/i18n/ru.json5]:
```suggestion
  "browse.empty": "Ничего не найдено.",
```

File: src/assets/i18n/ru.json5 [hunk 7]
```diff
@@ -1549,257 +1549,208 @@
...
   
   // "browse.metadata.pjtitle" : "Title",
-  // TODO New key - Add a translation
-  "browse.metadata.pjtitle" : "Title",
+  "browse.metadata.pjtitle" : "Заглавие",
   
   // "browse.metadata.rodept": "Department",
-  // TODO New key - Add a translation
-  "browse.metadata.rodept": "Department",
+  "browse.metadata.rodept": "Организация",
```
Comment (apalkoff) [inline src/assets/i18n/ru.json5]:
```suggestion
  "browse.metadata.rodept": "Подразделение",
```

File: src/assets/i18n/ru.json5 [hunk 8]
```diff
@@ -1549,257 +1549,208 @@
...
   
   // "browse.metadata.rpdept": "Department",
-  // TODO New key - Add a translation
-  "browse.metadata.rpdept": "Department",
+  "browse.metadata.rpdept": "Организация",
```
Comment (apalkoff) [inline src/assets/i18n/ru.json5]:
```suggestion
  "browse.metadata.rpdept": "Подразделение",
```

File: src/assets/i18n/ru.json5 [hunk 9]
```diff
@@ -1549,257 +1549,208 @@
...
   
   // "browse.metadata.ouname.breadcrumbs" : "Browse by Name",
-  // TODO New key - Add a translation
-  "browse.metadata.ouname.breadcrumbs" : "Browse by Name",
+  "browse.metadata.ouname.breadcrumbs" : "Поиск по Имени",
   
   // "browse.metadata.pjtitle.breadcrumbs" : "Browse by Title",
-  // TODO New key - Add a translation
-  "browse.metadata.pjtitle.breadcrumbs" : "Browse by Title",
+  // необходимо понять, что за pjtitle и eqtitle
```
Comment (apalkoff) [inline src/assets/i18n/ru.json5:1649]: вероятно pj — printed journal

File: src/assets/i18n/ru.json5 [hunk 10]
```diff
@@ -1549,257 +1549,208 @@
...
-  "browse.next.button": "Next",
+  "browse.next.button": "Далее",
   
   // "browse.previous.button": "Previous",
-  // TODO New key - Add a translation
-  "browse.previous.button": "Previous",
+  "browse.previous.button": "Назад",
   
   // "browse.startsWith.choose_start": "(Choose start)",
+  // Почему в скобках?
```
Comment (apalkoff) [inline src/assets/i18n/ru.json5:1679]: узнаем потом

File: src/assets/i18n/ru.json5 [hunk 11]
```diff
@@ -1549,257 +1549,208 @@
...
   // TODO New key - Add a translation
   "browse.startsWith.choose_start": "(Choose start)",
   
   // "browse.startsWith.choose_year": "(Choose year)",
+  // Почему в скобках?
```
Comment (apalkoff) [inline src/assets/i18n/ru.json5:1684]: возможно это надписи для пустых полей при заполнении. ну знаешь такие серенькие бывают.

File: src/assets/i18n/ru.json5 [hunk 12]
```diff
@@ -1549,257 +1549,208 @@
...
   
   // "browse.startsWith.submit": "Browse",
-  // TODO New key - Add a translation
-  "browse.startsWith.submit": "Browse",
+  "browse.startsWith.submit": "Поиск",
   
   // "browse.startsWith.type_date": "Or type in a date (year-month) and click 'Browse'",
-  // TODO New key - Add a translation
-  "browse.startsWith.type_date": "Or type in a date (year-month) and click 'Browse'",
+  "browse.startsWith.type_date": "Или введите дату (год-месяц) и нажмите 'Поиск'",
```
Comment (apalkoff) [inline src/assets/i18n/ru.json5:1744]: а может всё же и поиск

**Review by apalkoff — CHANGES_REQUESTED** [review]
Надо поправить

**Comment by mamedzuka** [comment]
ну, я поменял, теперь буду по-русски писать - 1 заглавная буква
надо будет челам, которые работать тоже будут, про общий интерфейс написать
(ну про 1 заглавную букву, про ты-вы и прочая)

**Review by apalkoff — APPROVED** [review]
ok

**Comment by mamedzuka** [comment]
а чо Merge сделан или потом делать будем?

**Review by apalkoff — APPROVED** [review]
ok

File: src/assets/i18n/ru.json5 (hunk 4 above)
Comment (apalkoff) [reply src/assets/i18n/ru.json5]: ну так что же?

**Review by apalkoff — COMMENTED** [review]
ну чего с этим делать? вливать?

**Comment by mamedzuka** [comment]
этот давно мерджить надо
collection держим
browse мерджим

**Pull request merged**